workers/merkle-tree-worker/src/interface.ts

```diff
@@ -0,0 +1,12 @@
+import type { generate_merkle_proof } from "@anonklub/merkle-tree-wasm";
+
+export type GenerateMerkleProofFn = (leaves: string[], leaf: string, depth: number) => Promise<Uint8Array>;
+
+export interface IMerkleTreeWorker {
+    prepare: () => void;
+    generateMerkleProof: GenerateMerkleProofFn;
+}
+
+export interface IMerkleTreeWasm {
+    generate_merkle_proof: typeof generate_merkle_proof
```

Comment:
I think this will trigger an linting error (or only a warning) about using snake_case instead of camelCase.
This is a requirement for the wasm binding to work right? (The rust function using snake_case and being named `generate_merkle_proof`)
If yes you may ned to add an eslint ignore statement

Reply:
The linter didn't print any errors regarding that case.